js/cryptocom.js

```diff
@@ -0,0 +1,1037 @@
+        //              {
+        //                  "currency": "CRO",
+        //                  "create_time": 1615886328000,
+        //                  "id": "12345",
+        //                  "address": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+        //                  "status": "1",
+        //                  "network": "CRO"
+        //              },
+        //              {
+        //                  "currency": "CRO",
+        //                  "create_time": 1615886332000,
+        //                  "id": "12346",
+        //                  "address": "yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy",
+        //                  "status": "1",
+        //                  "network": "ETH"
+        //              }
+        //          ]
+        //    }
+        // }
+        const data = this.safeValue (response, 'result', {});
+        const address = this.safeValue (data, 'deposit_address_list', []);
+        const addressList = [];
+        for (let i = 0; i < address.length; i++) {
+            addressList.push (address[i].address);
+        }
+        this.checkAddress (addressList[0]);
+        return {
+            'currency': code,
+            'address': addressList[0],
+            'info': data,
+        };
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let currency = undefined;
+        const request = {};
+        if (code !== undefined) {
+            currency = this.currency (code);
+            request['currency'] = currency['id'];
+        }
+        if (since !== undefined) {
+            request['start_ts'] = since;
+            // max 3 months range https://exchange-docs.crypto.com/spot/index.html#private-get-withdrawal-history
+            request['endTime'] = this.sum (since, 7776000000);
+        }
+        if (limit !== undefined) {
+            request['end_ts'] = limit;
+        }
+        const response = await this.privatePostPrivateGetDepositHistory (this.extend (request, params));
+        // {
+        //     "id": 11,
+        //     "method": "private/get-deposit-history",
+        //     "code": 0,
+        //     "result": {
+        //       "deposit_list": [
+        //         {
+        //           "currency": "XRP",
+        //           "fee": 1.0,
+        //           "create_time": 1607063412000,
+        //           "id": "2220",
+        //           "update_time": 1607063460000,
+        //           "amount": 100,
+        //           "address": "2NBqqD5GRJ8wHy1PYyCXTe9ke5226FhavBf?1234567890",
+        //           "status": "1"
+        //         }
+        //       ]
+        //     }
+        // }
+        const data = this.safeValue (response, 'result', {});
+        const depositList = this.safeValue (data, 'deposit_list', []);
+        return this.parseTransactions (depositList, currency, since, limit);
+    }
+
```

Comment:
fetchWithdrawals is missing

Reply:
yep currently  not implemented but it can be added later